src/components/authentication/CompletePasswordResetForm.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { connect } from 'react-redux';
+import { resetPassword } from '../../store/actions/authentication';
+
+class PasswordResetForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: '',
+    };
+
+    this.onChange = this.onChange.bind(this);
+    this.onSubmit = this.onSubmit.bind(this);
+  }
+
+  onChange(e) {
+    this.setState({ [e.target.name]: e.target.value });
+  }
+
+  onSubmit(e) {
+    e.preventDefault();
+    const token = new URLSearchParams(window.location.search).get('token');
+    const body = {
+      password: this.state.password,
+      token,
+    };
+    // console.log(this.props)
```

Comment:
Do you still need the ` console.log` ??

Reply:
This was probably the first commit, I did change this.